add Data server landing page as related information type (#282)
metno/mmd: doc/ch04-md-contrvocabs.adoc

facility following OGC Observations and Measurements or WMO Integrated
Global Observing System approach.
|Extended metadata |Additional unspecified metadata on the data. 

> |Data server landing page|The URL to access an application server (e.g. THREDDS, Hyrax and ERDDAP) landing page or catalog.

Reviewer: ```suggestion
|Data server landing page|The URL of an application server (e.g. THREDDS, Hyrax and ERDDAP) landing page or catalog.
```
